test/testHelpers.hpp

```diff
@@ -71,6 +71,46 @@ void readTests(const std::string &FileName, std::vector<af::dim4> &inputDims,
     }
 }
 
+template<typename outType>
+void readTestsOutputOnly(const std::string &FileName, std::vector<af::dim4> &inputDims,
```

Comment:
Not if I want to have the input dimensions, for gaussiankernel() I can still use the test input dimensions as the arguments passed to the method.